composer.json

```diff
@@ -7,7 +7,7 @@
     "prefer-stable": true,
     "require": {
         "php": "^7.2 || ^8.0",
-        "phpstan/phpstan": "^1.1",
+        "phpstan/phpstan": "^1.7",
```

Comment:
Is there a reason why we have to raise to version 1.7?
I'm fine with it but maybe it would be better to only do it if required if our code is still compatible. I had to raise to 1.6 anyways (See #57 ) for my upcoming changes in #48

Reply:
Yes, i would like to use the InitializerExprTypeResolver introduced in version 1.7.x https://github.com/phpstan/phpstan-src/pull/1276